include/mesh/exodusII_io.h

```diff
@@ -313,6 +313,16 @@ class ExodusII_IO : public MeshInput<MeshBase>,
                        const Real time,
                        const std::set<std::string> * system_names=nullptr);
 
+  /**
+   * Writes out the solution for no specific time or timestep.
+   * \param fname Name of the file to write to
+   * \param es EquationSystems object which contains the solution vector.
+   * \param system_names Optional list of systems to write solutions for.
+   */
+  virtual void write_equation_systems (const std::string & fname,
+                                       const EquationSystems & es,
+                                       const std::set<std::string> * system_names=nullptr);
```

Comment:
This should be marked `override`, there are some compilers that warn when that's missing IIRC.

Reply:
Quite right. :)